src/test/java/com/jfrog/ide/idea/scan/SourceCodeManagerTest.java

```diff
@@ -0,0 +1,39 @@
+package com.jfrog.ide.idea.scan;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+import static com.jfrog.ide.idea.scan.SourceCodeScannerManager.convertToSkippedFolders;
+
+@RunWith(Parameterized.class)
+public class SourceCodeManagerTest {
+
+    @Parameterized.Parameters
+    public static Collection<Object[]> data() {
+        return Arrays.asList(new Object[][]{
+                {"", new String[]{}},
+                {"**/*{.idea}*", new String[]{"**/*.idea*/**"}},
+                {"**/*.idea*", new String[]{"**/*.idea*/**"}},
+                {"**/*{.idea,test,node_modules}*", new String[]{"**/*.idea*/**", "**/*test*/**", "**/*node_modules*/**"}},
```

Comment:
It might be a good idea (if possible) to also test invalid patterns.

Reply:
We already test invalid patterns as part of the ConfigVerificationUtilsNegativeTest :)